cli/installer/docker_compose.go

```diff
@@ -325,6 +325,9 @@ func getCompleteProject(ui cliUI.UI, config configuration) *types.Project {
 	project, err := loader.Load(types.ConfigDetails{
 		WorkingDir:  workingDir,
 		ConfigFiles: configFiles,
+		Environment: map[string]string{
+			"TRACETEST_DEV": "",
+		},
```

Comment:
This is the main change of this PR to avoid warnings. We use the same set of tools as `docker-compose`, which emits a warning on `loader.Load` even if we are not running the containers.